source/includes/steps-start-sharded-cluster.yaml

```diff
@@ -0,0 +1,48 @@
+title: "Start config servers."
+stepnum: 1
+ref: start-config-servers
+content: |
+
+   Start the :binary:`~bin.mongod` instance for each of the
+   :doc:`config servers </core/sharded-cluster-config-servers>`, specifying
+   ``--configsvr``, ``--replSet``, ``--bind_ip``, and other options as
+   appropriate to your deployment.
+
+   .. code-block:: shell
+
+      mongod --configsvr --replSet <replicaSetName> --bind_ip localhost,<hostname(s)|ip address(es)>
+
+---
+title: "Start shards."
+stepnum: 2
+ref: start-shards
+content: |
+
+   Start the :binary:`~bin.mongod` instance for each replica set member of
+   each :doc:`shard </core/sharded-cluster-shards>`, adding options as
+   appropriate for your deployment.
+
+---
+title: "Start :binary:`mongos <bin.mongos>` routers."
+stepnum: 3
+ref: start-mongos
+content: |
+
+   Start each :binary:`mongos <bin.mongos>` router, adding options as
+   appropriate for your deployment:
+
+   .. code-block:: shell
+
+      mongos --config /path/to/config/mongos.conf
+
+   If the CSRS replica set name or any member hostname changed, update the
```

Comment:
Maybe it doesn't apply to a restart procedure, but for backup/restore there were a [bunch](https://docs.mongodb.com/manual/tutorial/restore-sharded-cluster/#for-any-planned-or-completed-shard-hostname-or-replica-set-name-changes-update-the-metadata-in-config-shards) of extra bits required to deal with changed hostnames of CSRS or shard members.

Reply:
Also we may need to specify `CSRS replica set member hostname`, so users can't interpret this as  any arbitrary member hostname.

Reply:
That's a good point. The purpose of the procedure is mainly to state the correct order for stopping and starting a sharded cluster, so I think it makes sense just to remove the mention of changed names.